api/accounts/delete.ts

```diff
@@ -0,0 +1,17 @@
+import handler from '../handler';
+import withConnection from '../db';
+import * as auth from '../authenticate';
+import * as errors from '../errors';
+import { Client } from 'pg';
+
+
+// Expects patientId
+// Returns null value
+export default handler(async (req, userId) => {
+    return withConnection(async (db: Client) => {
+        let deleteResults = await db.query('DELETE FROM authority WHERE authority.username = $1', [req.params.accountId]);
```

Comment:
This requires a username, but is given an authority id.
```suggestion
        let deleteResults = await db.query('DELETE FROM authority WHERE authority.accountId = $1', [req.params.accountId]);
```